doc/api/corepack.md

```diff
@@ -0,0 +1,95 @@
+# Corepack
+
+<!-- introduced_in=REPLACEME -->
+<!-- type=misc -->
+
+> Stability: 1 - Experimental
+
+_[Corepack][]_ is an experimental tool shipped with Node.js to help with
+managing versions of your package managers. It exposes binary proxies for
+[Yarn][] and [pnpm][] that, when called, locate the package managers configured
+for your local projects, transparently install them, and finally run them
+without requiring explicit user interaction.
+
+This feature allows you to make sure that everyone in your team is using
+exactly the package manager version you want them to without having to check
+them into your repository or managing complex migrations across developers'
+systems.
+
+## Workflows
+
+### Enabling the feature
+
+Due to its experimental status Corepack currently needs to be explicitly
+enabled to have any effect. To do that simply run [`corepack enable`][], which
+will set up the symlinks in your environment, next to the `node` binary
+(and overwrite the existing symlinks if necessary).
+
+From this point forward, any call to the `yarn`, `pnpm`, or `pnpx` binaries
+will work without further setup. Should you experience a problem, just run
+[`corepack disable`][] to remove the proxies from your system (and consider
+opening up an issue on the [Corepack repository][] to let us know).
+
+### Configuring a package
+
+The Corepack proxies will find the closest [`package.json`][] files in your
+directory hierarchy to extract their [`"packageManager"`][] property.
+
+If the requested package manager is [supported][], Corepack will make sure that
+all calls to the relevant binaries are run against the requested version.
```

Comment:
What if the requested version doesn't exist? Does it fall back to the latest known stable version, or does it fail with an error message? Either way, that would be interesting to have it documented here.

Reply:
```suggestion
If the value corresponds to a [supported package manager][], Corepack will make sure
all calls to the relevant binaries are run against the requested version.
```